modxcms/revolution
# Css fix

### What does it do?
Fix css

### Why is it needed?
Some days  ago @ilyautkin made fixes for mobile optimization. Somehow manager css files were broken. 
I run grunt and generate new css files

### How to test
Install current modx version. Check manager. 
replace CSS files with files from this branch. Check 

 Before:
![error](https://user-images.githubusercontent.com/4846064/128140557-7c665018-f255-4e67-a7ef-128269d4eee6.jpg)
After:
![2222](https://user-images.githubusercontent.com/4846064/128147358-a11252f0-b3b0-4010-8946-723b4ca66acc.jpg)
